pulp_ansible/tests/functional/api/collection/test_sync.py

```diff
@@ -280,3 +281,64 @@ def test_v2_simple_dependency(self):
 
         content = self.cv_api.list(repository_version=f"{repo.pulp_href}versions/1/", name="posix")
         self.assertNotEqual(content.count, 0)
+
+
+@pytest.mark.skip("Skipped until fixture metadata has a published date")
```

Comment:
Turns out galaxy does not provide the published info. So it cannot be tested this way. I would still keep the test, and we can maybe find another source to test this.
OTOH, is the "metadata" a specific file? We could just save it's digest and compare that...

Reply:
If you wanted to test this you would have to perform a Pulp to Pulp sync. The published metadata is only apart of the v3 api.